src/content/getting-started/custom-post-types.mdx

```diff
@@ -16,6 +16,10 @@ This is an example of registering a new “docs” post_type and enabling GraphQ
 ```php
 add_action( 'init', function() {
    register_post_type( 'docs', [
+      'show_ui' => true,
+      'labels'  => [
+	'menu_name' => 'Docs'
```

Comment:
@jmau111 if we want to include labels here, we should at least include them wrapped with translations, like `__( 'Docs, 'your-textdomain' ),`

I'm ok adding a few fields like this for example sake, but if users are trying to register a post type, and don't fully understand the core post type registry options, they should probably get familiar with them first. 

I'm thinking we should include some language around that and a link to the WordPress core docs on registering post types.

Reply:
@jasonbahl that's a good idea you're right.